<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="1">
    <widget class="QTabWidget" name="TabsWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Terminal">
      <attribute name="title">
       <string>Terminal</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" rowspan="2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Terminal Characteristics</string>
             </property>
             <widget class="QLabel" name="label_7">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>138</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Model</string>
              </property>
             </widget>
             <widget class="QComboBox" name="terminalType">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>50</y>
                <width>138</width>
                <height>31</height>
               </rect>
              </property>
              <property name="placeholderText">
               <string/>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>Model 2 (24x80)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Model 3 (32x80)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Model 4 (43x80)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Model 5 (27x132)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dynamic</string>
               </property>
              </item>
             </widget>
             <widget class="QLabel" name="label_6">
              <property name="geometry">
               <rect>
                <x>90</x>
                <y>130</y>
                <width>8</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>x</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
             <widget class="QLabel" name="label_5">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>90</y>
                <width>138</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Dynamic Size</string>
              </property>
             </widget>
             <widget class="QWidget" name="layoutWidget">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>110</y>
                <width>60</width>
                <height>56</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Rows</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="terminalRows">
                 <property name="maximum">
                  <number>999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="layoutWidget">
              <property name="geometry">
               <rect>
                <x>100</x>
                <y>110</y>
                <width>60</width>
                <height>56</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Columns</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="terminalCols">
                 <property name="maximum">
                  <number>999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Cursor Settings</string>
             </property>
             <widget class="QCheckBox" name="cursorColour">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>53</y>
                <width>836</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>Inherit colour of underlying character</string>
              </property>
             </widget>
             <widget class="QSlider" name="cursorBlinkSpeed">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>130</y>
                <width>151</width>
                <height>20</height>
               </rect>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>4</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBelow</enum>
              </property>
              <property name="tickInterval">
               <number>1</number>
              </property>
             </widget>
             <widget class="QCheckBox" name="cursorBlink">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>80</y>
                <width>836</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>Blink</string>
              </property>
             </widget>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>110</y>
                <width>91</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Blink Speed</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>150</y>
                <width>31</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Slow</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>140</x>
                <y>150</y>
                <width>31</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Fast</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="FontTab">
      <attribute name="title">
       <string>Font</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="FontLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5"/>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>Font Tweaks</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QComboBox" name="comboBox">
                <item>
                 <property name="text">
                  <string>Leave Zero As Font</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Add a dot to Zero</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Add a slash to zero</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="FontScaling">
                <property name="text">
                 <string>Scale Font to Window</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Colours">
      <attribute name="title">
       <string>Colours</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Base Colours</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Unprotected</string>
            </property>
            <property name="buddy">
             <cstring>pushButton</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">background-color: #ff00ff</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Protected</string>
            </property>
            <property name="buddy">
             <cstring>pushButton_2</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Unprotected, Intensified</string>
            </property>
            <property name="buddy">
             <cstring>pushButton_3</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Unprotected, Intensified</string>
            </property>
            <property name="buddy">
             <cstring>pushButton_4</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Extended Colours</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Black</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="colourBlack">
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Blue</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="colourBlue">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Red</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="colourRed">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Magenta (Pink)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="colourPink">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Green</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="colourGreen">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Cyan (Turquoise)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="colourTurq">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Yellow</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="colourYellow">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>White</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="colourWhite">
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="ButtonLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>terminalType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Settings</receiver>
   <slot>changeModel(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colourBlue</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>selectColour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colourBlack</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>selectColour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeModel(int)</slot>
  <slot>changeBlink(int)</slot>
  <slot>changeBlinkSpeed(int)</slot>
  <slot>selectColour()</slot>
 </slots>
</ui>
